installation/components/setup_hifiberry.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# This script follows the official HiFiBerry documentation
+# https://www.hifiberry.com/docs/software/configuring-linux-3-18-x/
+
+# Check if the script is run as root
+if [ "$(id -u)" != "0" ]; then
+   echo "This script must be run as root" 1>&2
+   exit 1
+fi
+
+# List of HiFiBerry boards
+echo "
+Select your HiFiBerry board:
+1) DAC for Raspberry Pi 1/DAC+ Light/DAC Zero/MiniAmp/Beocreate/DAC+ DSP/DAC+ RTC
+2) DAC+ Standard/Pro/Amp2
+3) DAC2 HD
+4) DAC+ ADC
+5) DAC+ ADC Pro
+6) Digi+
+7) Digi+ Pro
+8) Amp+ (not Amp2)
+9) Amp3"
+read -p "Enter your choice (1-9): " choice
+
+enable_hifiberry() {
+    echo "Enabling HiFiBerry board..."
+    grep -qxF "dtoverlay=$1" /boot/config.txt || echo "dtoverlay=$1" >> /boot/config.txt
```

Comment:
Does it still work if this is run multiple times, but with a different selection?
Looks like then this entry is just added. Could this cause problems?

Possible case if I used the wrong choice but only found out after further testing?

Reply:
Agreed. Will fix in the next iteration!